cloudinit/config/cc_snap.py

```diff
@@ -105,6 +105,7 @@
         ),
     ],
     "frequency": PER_INSTANCE,
+    "skip_by_schema": ["snap"],
```

Comment:
I agree. This renaming does also improve developer experience, as the old name allowed double negation.

Additionally, I have renamed `_is_inapplicable` to `_is_active` to avoid double negation.

Reply:
looks good.